ab_tool/canvas.py

```diff
@@ -90,6 +92,19 @@ def _get_installed_intervention_point_urls(self):
         return installed_intervention_point_urls
 
 
+def get_unsorted_students(request, experiment):
+    request_context = get_canvas_request_context(request)
+    course_id = get_lti_param(request, "custom_canvas_course_id")
+    try:
+        enrollments = list_users_in_course_users(
+                request_context, course_id, None, enrollment_type="student").json()
+    except RequestException as exception:
+        handle_canvas_error(exception)
+    existing_student_ids = set(s.id for s in experiment.students.all())
+    return [{"student_id": i["sis_user_id"], "lis_person_sourcedid": i["sis_user_id"]}
```

Comment:
Not sure I understand why the sis_user_id is being used as a value for both the student_id and lis_person_sourcedid.  Is an `sis_user_id` guaranteed to be present?  Why not just return the whole student record you're getting from the API and actually use the `sis_user_id` field in the new csv methods you've added in place of the proxy values you're using here?  At the very least you should document this method so that if others want to use it they will understand that the keys they're getting are tied to the sis id.